### ISSUE TITLE
Problem starting due to sendDataValue

### ISSUE THREAD

**Issue by zagar** [jvmahon/Homebridge-HomeSeer4#75]
I installed homebridge and homebridge-homeseer4 on a RPI 3 running buster. There isn't much else running on this system.

**npm -g list --depth 0**
/usr/local/lib
tqq homebridge@1.0.4
tqq homebridge-config-ui-x@4.19.0
tqq homebridge-dummy@0.4.1
tqq homebridge-homeseer4@1.0.0
mqq npm@6.14.5

**node -v**
v10.19.0

Here's the relevant portion of config.json:
```
{
  "platform": "HomeSeer",
  "name": "HomeSeer",
  "host": "***",
  "ASCIIPort": 11000,
  "login": "***",
  "password": "***",
  "Lightbulbs": [
    88
  ]
}
```

homebridge will not start with the current plugin and gives this error:

```
[5/14/2020, 11:48:16 PM] ERROR LOADING PLUGIN homebridge-homeseer4:
[5/14/2020, 11:48:16 PM] /usr/local/lib/node_modules/homebridge-homeseer4/lib/HomeSeerSystemObject.js:168
        sendDataValue = function(ref, newValue, sendOnlyOnChange = true) 
                      ^

SyntaxError: Unexpected token =
    at Module._compile (internal/modules/cjs/loader.js:723:23)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/homebridge-homeseer4/index.js:16:24)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
[5/14/2020, 11:48:16 PM] ====================
```

If I manually edit HomeSeerSystemObject.js and change line 186 to remove " = function" to change it to:

```
sendDataValue(ref, newValue, sendOnlyOnChange = true)
```

to align its setup to match the other functions, homebridge will start and everything works.

**Comment by jvmahon** [comment]
Thanks for the info. I'll make that change and test it out this weekend.

**Comment by chriszagaremcc** [comment]
The homeseer 1.0.1 update addressed this issue and also the homebridge 1.1.0 error perfectly on my test system. Thanks!
